cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -808,6 +816,43 @@ instance ToObject NtN.HandshakeTr where
              , "bearer" .= show b
              , "event" .= show ev ]
 
+instance ( Mock.SimpleCrypto c
+         , Typeable a)
+ => ToObject (Header (Mock.SimpleBlock c a)) where
+  toObject _verb b =
+    mkObject $
+        [ "kind" .= String "SimpleBlockHeader"
+        , "hash" .= condense (blockHash b)
+        , "prevhash" .= condense (blockPrevHash b)
+        , "slotNo" .= condense (blockSlot b)
+        , "blockNo" .= condense (blockNo b) ]
+
+instance ToObject (Header ByronBlock) where
+  toObject _verb b =
+    mkObject $
+        [ "kind" .= String kind
+        , "hash" .= condense (blockHash b)
+        , "prevhash" .= condense (blockPrevHash b)
+        , "slotNo" .= condense (blockSlot b)
+        , "blockNo" .= condense (blockNo b)
+        ] <>
+        case view of
+          Nothing -> []
+          Just (diffic, deleg) ->
```

Comment:
The `diffic` is unused (doesn't this cause a build failure due to warnings as errors?). The `view` below can be simplified by not calculating it. And perhaps `view` can be renamed to something like `signer` or `delegate`.

Reply:
Inlined it all.